db/migrate/20191011142024_remove_auth_by_pass_i_ds_from_access_limited.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveAuthByPassIDsFromAccessLimited < Mongoid::Migration
+  def self.up
+    ContentItem
+      .where(:access_limited.nin => [{}, nil])
+      .each { |ci| ci.unset("access_limited.auth_bypass_ids") }
+  end
+
+  def self.down
+  end
+end
```

Comment:
It might be an idea to hold fire on having this migration in here since this needs to occur after this code is running on production.

As it's a bit of a do or die moment we probably want to have it kept granular.

Reply:
I'll cherry-pick this out into a separate PR.

Reply:
https://github.com/alphagov/content-store/pull/636